test/Npgsql.Tests/MultipleHostsTests.cs

```diff
@@ -312,7 +387,7 @@ public void Cluster_state_cache_basic()
 
         // This is the only test in this class which actually connects to PostgreSQL (the others use the PostgreSQL mock)
         [Test, Timeout(10000), NonParallelizable]
-        public void IntegrationTest()
+        public void IntegrationTest([Values(true, false)] bool withLoadBalancing)
```

Comment:
nit: can omit the true and false, [Values] automatically goes through all the possible values of bools (and enums)